Acknowledgment_guidelines.md

```diff
@@ -1,8 +1,16 @@
 # Acknowledging KBase support
 
-The following guidelines should be considered when producing materials about other work that may have used KBase resources.  This includes peer-reviewed publications submitted to journals, conference presentations (peer-reviewed or not), websites, etc. When in doubt, please contact [FIXME: who?] for further guidance.
+## Formal statements for acknowledgment and process
+
+When writing a paper that cites KBase in any way, you should acknowledge KBase funding as follows: “This work supported by the Office of Science, Office of Biological and Environmental Research, of the US Department of Energy under Contract DE-AC02-06CH1135, as part of the DOE Systems Biology Knowledgebase.”
```

Comment:
As per @bobcottingham, the funding acknowledgement should be:
This work is supported as part of the Genomic Sciences Program DOE Systems Biology Knowledgebase (KBase) funded by the U.S. Department of Energy, Office of Science, Office of Biological and Environmental Research under Award Numbers DE-AC02-05CH11231, DE-AC02-06CH11357, DE-AC05-00OR22725, and DE-AC02-98CH10886.